README.md

```diff
@@ -0,0 +1,10 @@
+## Font Sizes
```

Comment:
Please give a small description as well regarding what problem actually this code is solving for Devs.